### PR TITLE
Add "Needs: Breakdown" to issues templates

### PR DESCRIPTION
**Pull request by mekarpeles**
Project management change informed by discussions with @mekarpeles @scottbarnes @jimchamp and @cdrini during staff calls.

https://github.com/mekarpeles/openlibrary/issues/new?assignees=&labels=Type%3A+Bug%2CNeeds%3A+Triage%2CNeeds%3A+Lead%2CNeeds%3A+Breakdown&projects=&template=bug_report.yaml

## Bugs

Screenshot 2024-08-19 at 9 45 35 AM

Screenshot 2024-08-19 at 9 46 47 AM

## Features

https://github.com/mekarpeles/openlibrary/issues/new?assignees=&labels=Type%3A+Feature+Request%2CNeeds%3A+Triage%2CNeeds%3A+Lead%2CNeeds%3A+Breakdown&projects=&template=feature_request.yaml

Screenshot 2024-08-19 at 10 38 41 AM

Screenshot 2024-08-19 at 10 39 36 AM

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 1]
```diff
@@ -1,50 +1,51 @@
...
-      label: Proposal & Constraints
-      description: Describe the proposed solution and possible constraints
+      label: Justification
+      description: Help us understand, what problem(s) the proposal addresses, what is the predicted impact and how might it be measured, which audiences are served, are there precedents for the feature, and any other context we might consider.
       value: |
-        ### What is the proposed solution / implementation?
+        <!-- Problem: What problem does this proposal address and for what audience(s)? -->
 
-        ### Is there a precedent of this approach succeeding elsewhere?
+        <!-- Impact: What is the predicted impact, how will we measure impact, and what does success look like? -->
```
Comment (mekarpeles) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:21]:
```suggestion
        <!-- Impact: What's the predicted impact, how to measure, & what does success look like? -->
```

File: .github/ISSUE_TEMPLATE/feature_request.yaml [hunk 2]
```diff
@@ -1,50 +1,51 @@
...
     attributes:
-      label: Leads
+      label: Breakdown
       description: |
-        Please leave this section blank for maintainers if you are unsure what to enter
+        Instructions and details for completing this feature. Please leave this section blank for leads/maintainers if you are unsure what to enter
       value: |
-        ### Related files
+        #### Requirements Checklist
+        <!-- What are the steps, functional pieces, or requirements that need to be met to satisfy this proposal? -->
```
Comment (mekarpeles) [inline .github/ISSUE_TEMPLATE/feature_request.yaml:32]:
```suggestion
        <!-- What steps, functional pieces, or requirements need to be met? -->
```

**Comment by mekarpeles** [comment]
please squash on merge

**Comment by mekarpeles** [comment]
TODO:
%cursor%, consistent IMPORTANT, expected v. actual, *'s for breakdown fmt

**Review by jimchamp — APPROVED** [review]

**Pull request merged**